audiobookshelf/docker-compose.instance.yaml

```diff
@@ -30,3 +33,14 @@ services:
 
       #! Apply all middlewares (do this at the end!)
       - "traefik.http.routers.(@= router @).middlewares=(@= ','.join(enabled_middlewares) @)"
+
+      #! Homepage labels for auto-discovery and config:
+      - "homepage.group=Group A"
+      - "homepage.name=Audiobookshelf"
+      - "homepage.icon=audiobookshelf.png"
+      - "homepage.href=http://(@= traefik_host @)"
+      - "homepage.description=Self-hosted audiobook and podcast server"
+      - "homepage.widget.type=audiobookshelf"
```

Comment:
I think this should be added conditionally if AUDIOBOOKSHELF_API_KEY is non-blank.
```
#@ if len(AUDIOBOOKSHELF_API_KEY):
...
#@ end
```